dev/benchmarks/test_apps/stocks/test/icon_color_test.dart

```diff
@@ -58,8 +60,9 @@ void main() {
     expect(find.text('Account Balance'), findsNothing);
 
     // drag the drawer out
-    final Offset left = Offset(0.0, (WidgetsBinding.instance.window.physicalSize / WidgetsBinding.instance.window.devicePixelRatio).height / 2.0);
-    final Offset right = Offset((WidgetsBinding.instance.window.physicalSize / WidgetsBinding.instance.window.devicePixelRatio).width, left.dy);
+    final ui.FlutterView view = tester.widget<View>(find.byType(View)).view;
```

Comment:
Should we add a helper to `WidgetTester` to get the current view?

Reply:
Paul is actually working on that as part of his change. Let me revert this file and then we can use that once it is ready (there are a few other places that I did not migrate because I was waiting for Paul's change).

Reply:
Feel free to just leave a TODO instead, your change is a good step in the right direction. Whatever you prefer :)